scripts/forseti-server/forseti_server_startup_script.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+exec > /tmp/deployment.log
+exec 2>&1
+
+# Ubuntu update.
+sudo apt-get update -y
+sudo apt-get upgrade -y
+sudo apt-get update && sudo apt-get --assume-yes install google-cloud-sdk
+
+# Env variables
+USER_HOME=/home/ubuntu
+
+# Install fluentd if necessary.
+FLUENTD=$(ls /usr/sbin/google-fluentd)
+if [ -z "$FLUENTD" ]; then
+      cd $USER_HOME
+      curl -sSO https://dl.google.com/cloudagents/install-logging-agent.sh
+      bash install-logging-agent.sh
+fi
+
+# Check whether Cloud SQL proxy is installed.
+CLOUD_SQL_PROXY=$(which cloud_sql_proxy)
+if [ -z "$CLOUD_SQL_PROXY" ]; then
+      cd $USER_HOME
+      wget https://dl.google.com/cloudsql/cloud_sql_proxy.${cloudsql_proxy_arch}
+      sudo mv cloud_sql_proxy.${cloudsql_proxy_arch} /usr/local/bin/cloud_sql_proxy
+      chmod +x /usr/local/bin/cloud_sql_proxy
+fi
+
+# Install Forseti Security.
+cd $USER_HOME
+rm -rf *forseti*
+
+# Download Forseti source code
+git clone ${forseti_repo_url}
```

Comment:
Instead of templating this file, can we template a config file and source it here?